components/authentication-page-template.tsx

```diff
@@ -18,7 +18,7 @@ const AuthenticationPageTemplate: React.FC<AuthenticationPageTemplateProps> = ({
   className = "",
   needGeneralStyles = true,
 }) => {
-  const generalClasses = `z-10 ${needGeneralStyles && "rounded-2xl border bg-white px-7 py-6 md:rounded-[1.5rem] md:px-11 md:pt-9 md:pb-11 min-w-[90vw] sm:min-w-0 max-w-[90vw] sm:max-w-[80vw] lg:max-w-[45rem]"}`;
+  const generalClasses = `z-10 w-full ${needGeneralStyles && "rounded-2xl border bg-white px-7 py-6 md:rounded-[1.5rem] md:px-11 md:pt-9 md:pb-11 "}`;
 
```

Comment:
is this styling update in the template maintain the UI design  for all the pages this component is used in?

Reply:
@imrankhan-stixor, didn't add any new style; just used the previous styles and placed them in the right spot.